compiler/plugins/target/AMD-AIE/iree-amd-aie/Transforms/Passes.cpp

```diff
@@ -65,6 +65,8 @@ void addMLIRAIRAIELoweringPasses(OpPassManager &passManager) {
   passManager.addPass(createEraseHALDescriptorTypeFromMemRefPass());
   passManager.addPass(createAMDAIEBridgeToAIRPass());
   passManager.addPass(memref::createFoldMemRefAliasOpsPass());
+  passManager.addPass(
+      mlir::iree_compiler::createIREEComprehensiveBufferizePass());
```

Comment:
Hi @newling - this perhaps won't be required since [e2e PR](https://github.com/nod-ai/iree-amd-aie/pull/70) is taking care of adding `createIREEComprehensiveBufferizePass` [here](https://github.com/nod-ai/iree-amd-aie/blob/afcccb558bf26d93b6a8ee4e363140ff949a40dc/compiler/plugins/target/AMD-AIE/iree-amd-aie/Transforms/Passes.cpp#L56-L63).

CC: @MaheshRavishankar